inspirehep/modules/workflows/utils.py

```diff
@@ -176,3 +177,41 @@ def download_file_to_workflow(workflow, name, url):
         if req.status_code == 200:
             workflow.files[name] = req.raw
             return workflow.files[name]
+
+
+def store_original_record_for_source(record_id, source, data):
+    """Store the original record for a given source.
+
+    Given a previously matched ``record_id`` (i.e. its UUID), the ``source``
```

Comment:
You are right. It's a bit sad that the original APIs called it `record_uuid` in the first place, because this is mixing up type with semantic. But too bad.